Modules/Numerics/Statistics/include/itkWeightedCovarianceSampleFilter.hxx

```diff
@@ -30,17 +30,6 @@ WeightedCovarianceSampleFilter<TSample>::WeightedCovarianceSampleFilter()
   this->ProcessObject::SetNthInput(1, nullptr);
 }
 
-template <typename TSample>
-void
-WeightedCovarianceSampleFilter<TSample>::PrintSelf(std::ostream & os, Indent indent) const
-{
-  Superclass::PrintSelf(os, indent);
-  // m_Weights
-  os << indent << "Weights: " << this->GetWeightsInput() << std::endl;
```

Comment:
Does the parent print this?

Reply:
Same comment as in https://github.com/InsightSoftwareConsortium/ITK/pull/3862/files/74c520d4231066d796ce6d83c5ba4e3d4134c6c2#r1068787233.